_data/TateWalker/TateWalker.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Tate Walker
+institution: Texas A&M University
+profile_pic: 0Y5A9019.jpg
```

Comment:
Incorrect file name, please do one of the following:
1. Change `0Y5A9019.jpg` to `square.jpeg` in the markdown file.
2. Rename `square.jpeg` to `0Y5A9019.jpg`.